packages/webapp/package.json

```diff
@@ -30,7 +30,6 @@
     "dayjs": "^1.10.7",
     "ethereum-bloom-filters": "^1.0.10",
     "ethers": "^5.5.1",
-    "history": "^5.0.1",
```

Comment:
Was this accidental inclusion or necessary for this PR?

Reply:
Netlify build results in error without this due to dependencies mixup problems. The issue is only observed on Netlify, builds locally without a problem.